handler.go

```diff
@@ -56,13 +56,13 @@ func (ca *Cagent) initHubHTTPClient() {
 
 func (ca *Cagent) TestHub() error {
 	if ca.Config.HubURL == "" {
-		return fmt.Errorf("please set the hub_url config param")
+		return fmt.Errorf("please fill config with hub_url, hub_user and hub_password from your Cloudradar account")
 	}
 
 	var u *url.URL
 	var err error
 	if u, err = url.Parse(ca.Config.HubURL); err != nil {
-		return fmt.Errorf("can't parse hub_url: %s", err.Error())
+		return fmt.Errorf("can't parse hub_url: %s. Make sure to put the right params from your Cloudradar account", err.Error())
```

Comment:
`params` => `url` maybe?

Reply:
if we got the error parsing hub_url it doesn't mean that user put the right hub_user and hub_password. So that's why I have mention `params`

Reply:
yes, but if `url.Parse(ca.Config.HubURL)` fails we know _exactly_ that the url is wrong right? So we can give a specific error and not just a generic one or is there something later in the flow I miss right now?

Reply:
the error starts with: `can't parse hub_url`

Reply:
ok, fine...